src/librustdoc/externalfiles.rs

```diff
@@ -37,14 +37,14 @@ impl ExternalHtml {
         let bc = format!(
             "{}{}",
             bc,
-            Markdown(&m_bc, &[], id_map, codes, edition, playground).to_string()
+            Markdown(&m_bc, &[], id_map, codes, edition, playground).into_string()
```

Comment:
Can this change be done in a separate PR? It seems unrelated to this one.

Reply:
Split into #74359. Will update this PR soon.